programs/client/Client.cpp

```diff
@@ -487,6 +488,51 @@ class Client : public Poco::Util::Application
     }
 #endif
 
+    /// Make query to get all server warnings
+    void loadWarningMessages(std::vector<String>& messages)
+    {
+        connection->sendQuery(connection_parameters.timeouts, "SELECT message FROM system.warnings", "" /* query_id */, QueryProcessingStage::Complete);
```

Comment:
> If the table does not exist on server or if query cannot execute by another reason, catch exception and ignore it.

It is for compatibility with old servers.

Reply:
Is it bad to have try-catch over all this function? (lines 618-632)

Reply:
It is alright.